src/component/feature/add-tag-dialog-component.jsx

```diff
@@ -0,0 +1,100 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+import { Button, Dialog, Textfield, Card, CardTitle, CardText, CardActions } from 'react-mdl';
```

Comment:
Ok, now i understand why you where asking about the dialog polyfill. I have on purpose not used Dialog from react-mdl, but instead used [react-modal](https://www.npmjs.com/package/react-modal) several places (e.g. add-variant.jsx).